homeassistant/components/homematicip_cloud/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  ## Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: |
+      Integration is push based.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: todo
+  test-before-setup: todo
+  unique-config-entry: done
+  ## Silver
+  action-exceptions: todo
+  config-entry-unloading: done
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow: todo
+  test-coverage: todo
+  ## Gold
+  devices: done
+  diagnostics: todo
+  discovery-update-info:
+    status: exempt
+    comment: |
+      The Integration must be configured with a config flow and does not support discovery.
+      Devices are added automatically.
+  discovery:
+    status: exempt
+    comment: |
+      The Integration must be configured with a config flow and does not support discovery.
+      Devices are added automatically.
```

Comment:
We can still try to recognize devices on the network, even though we don't connect directly to the device

Reply:
How do you mean accesspoint? That means they are on your own network right?

Reply:
okay you were completely right. I just tried to find accesspoints in my network with the zeroconf lib and found 1 of 2 accesspoints in my network. I should set these rules to todo and have a look in the future.

Reply:
Feel free to ask questions about this in the integration owners channel if you need some :D

I always love it when integrations enable it and just automatically pop up when needed